### PR TITLE
[10.0][MIG] account_mass_reconcile_ref_deep_search

### PR DESCRIPTION
**Pull request by grindtildeath**
Supersedes : #208

### CODE REVIEW

File: account_mass_reconcile_ref_deep_search/tests/test_account_reconcile_ref_deep_search.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                'quantity': 1.0,
+            })],
+            'name': 'test_deep_search'
+        })
+        cls.cust_invoice.action_invoice_open()
+
+    def test_account_reconcile_ref_deep_search(self):
+        self.assertEqual(self.cust_invoice.state, 'open')
+        bank_journal = self.env['account.journal'].search([
+            ('type', '=', 'sale')], limit=1)
```
Comment (sebalix) [inline account_mass_reconcile_ref_deep_search/tests/test_account_reconcile_ref_deep_search.py]: Can we not reuse the `sales_journal` defined in `setUpClass`?

**Review by sebalix — COMMENTED** [review]
Just a question, otherwise LGTM

File: account_mass_reconcile_ref_deep_search/tests/test_account_reconcile_ref_deep_search.py (hunk 1 above)
Comment (grindtildeath) [reply account_mass_reconcile_ref_deep_search/tests/test_account_reconcile_ref_deep_search.py]: Well spotted :+1: 
That should be a bank journal, but I'm searching for sale. :confounded: 
I'll fix it.

Comment (sebalix) [reply account_mass_reconcile_ref_deep_search/tests/test_account_reconcile_ref_deep_search.py]: Okay, so the same fix applies to #232 !

**Review by sebalix — APPROVED** [review]

**Review by simahawk — APPROVED** [review]
LG

File: account_mass_reconcile_ref_deep_search/static/description/index.html [hunk 2]
```diff
@@ -0,0 +1,422 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (Tonow-c2c) [inline account_mass_reconcile_ref_deep_search/static/description/index.html:1]: you can not split in two file .html + one .ccs ?

File: account_mass_reconcile_ref_deep_search/tests/test_account_reconcile_ref_deep_search.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                'account_id': account_revenue.id,
+                'price_unit': 1000.0,
+                'quantity': 1.0,
+            })],
+            'name': 'test_deep_search'
+        })
+        cls.cust_invoice.action_invoice_open()
+
+    def test_account_reconcile_ref_deep_search(self):
+        self.assertEqual(self.cust_invoice.state, 'open')
```
Comment (Tonow-c2c) [inline account_mass_reconcile_ref_deep_search/tests/test_account_reconcile_ref_deep_search.py:42]: maybe add docstring :wink:

File: account_mass_reconcile_ref_deep_search/static/description/index.html (hunk 2 above)
Comment (grindtildeath) [reply account_mass_reconcile_ref_deep_search/static/description/index.html:1]: This is generated by oca-gen-readme :wink:

Comment (Tonow-c2c) [reply account_mass_reconcile_ref_deep_search/static/description/index.html:1]: ok thanks for explanation

**Comment by emagdalenaC2i** [comment]
@pedrobaeza Should this be merged?
And should be closed #229 in favour of #232?

**Comment by pedrobaeza** [comment]
@simahawk @sebalix please decide

**Comment by grindtildeath** [comment]
@emagdalenaC2i @pedrobaeza 
This PR can be merged since it's for v10.0 and the superseded PR is closed.
For v11.0 #229 was superseded by #232 where I included the changes done here and those by @sebalix on #229 , so IMO #229 can be closed and we'd still need a couple more reviews on #232 to get it merged.

@sebalix can you confirm ?

**Comment by sebalix** [comment]
@grindtildeath yep, PR #229 closed in favor of #232.

**Comment by grindtildeath** [comment]
@pedrobaeza Can you please hit the green button here since this is v10.0 and the mig to v11.0 is merged ? :pray: :innocent:

**Pull request merged**